### PR TITLE
🌟Training - Event booking component 

### LINKED ISSUE #918: Training - Add event booking component
https://www.ssw.com.au/ssw/Events/Training/Angular-Superpowers-Tour.aspx

move to /training/angular-superpowers

Image

**Figure: Booking div on existing page**

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixes #918 

**Description:**
Created a new event booking component for the following Div.
image
**Figure: Existing Event Booking Component** 

**New component screenshots:**

image

**Figure: Full screen in Desktop view** 

image

**Figure: Full screen in Desktop view with 4 events** 

image

**Figure: Full screen in Desktop view with 5 events** 

image

**Figure: Full screen in Desktop view with 6 events** 

image

**Figure: Mobile view** 

image

**Figure: iPad or mid screen view**

### CODE REVIEW

File: components/training/eventBooking.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      <span className="font-bold capitalize">{event.city}</span>
+      <div className=" py-0.5 text-xs uppercase text-gray-500">
+        {" "}
+        {event.date && moment(event.date).format("Do (ddd) MMMM YYYY")}
+      </div>
+      <div className=" py-0.5 text-xs uppercase text-gray-500">
+        {EventModel.TIMINGS}
+      </div>
+      <div className="py-1">
+        <a
```
[the remark this answers is not in the document]
Comment (amankumarrr) [reply components/training/eventBooking.tsx]: resolved

File: components/training/eventBooking.tsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+      </span>
+      <div
+        className=" py-0.5 text-xs uppercase text-gray-500"
+        data-tina-field={tinaField(
+          schema.eventList[index],
+          eventBookingBlock.eventList.date
+        )}
+      >
+        {" "}
+        {event.date && moment(event.date).format("Do (ddd) MMMM YYYY")}
```
Comment (bradystroud) [inline components/training/eventBooking.tsx]: Try use DayJS - We already have it installed 

https://chat.openai.com/share/ca2eab25-4e1e-4688-9be5-43a179a0ec4f

**Review by bradystroud — CHANGES_REQUESTED** [review]
Looking good 🙂

**Comment by bradystroud** [comment]
@amankumarrr Thoughts on moving this stuff to the right on mobile

image

**Comment by amankumarrr** [comment]
> @amankumarrr Thoughts on moving this stuff to the right on mobile
> 
> image

Yes that would look good, do you want me to add that?

File: components/training/eventBooking.tsx (hunk 2 above)
Comment (amankumarrr) [reply components/training/eventBooking.tsx]: done - replaced it

**Review by bradystroud — APPROVED** [review]

**Pull request merged**